style.css

```diff
@@ -0,0 +1,280 @@
+*{
+	margin: 0;
+	padding: 0;
+}
+.navbar-brand{
+    font-size: 1.6rem !important;
+    font-family: 'Pacifico', cursive;
+}
+.navbar{
```

Comment:
Please include one line of space between CSS rules/blocks for better readability. Check out Prettier.